client/derive_config.ts

```diff
@@ -17,7 +17,12 @@ export function deriveConfig(conf: ClientConfig = {}): Doc {
     const got: Doc = get({ profile: _conf.profile || "default" });
 
     if (typeof _conf.credentials !== "function") {
-      _conf.credentials = { ...got.credentials, ..._conf.credentials };
+      _conf.credentials = {
+        ...got.credentials,
+        ..._conf.credentials,
+        accessKeyId: got.accessKeyId,
+        secretAccessKey: got.secretAccessKey
+      };
```

Comment:
I believe we also need the SessionToken here too, agreed?

Reply:
yea, session token should be passed